packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/dateHeaderText.tsx

```diff
@@ -17,7 +17,7 @@ export const viewFunction = ({
     {
       splitText
         // eslint-disable-next-line react/jsx-key
-        ? textParts.map((part) => (<span className="dx-scheduler-header-panel-cell-date">{part}</span>))
+        ? textParts.map((part) => (<div className="dx-scheduler-header-panel-cell-date"><span>{part}</span></div>))
```

Comment:
What a reason change tag to <div />?

Reply:
I think difference in display style: was inline, became block.

Reply:
The following style changes the span element's display style to `block`: [_indes.scss:229](https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/scss/widgets/fluent/scheduler/_index.scss#L229).

The current solution requires changing the element's display style to `inline-flex`: [_index.scss:233](https://github.com/DevExpress/DevExtreme/pull/25558/files#diff-7dd61f89332668e9ffe9f2c7d6f9d8c61ff124eceab037c8783da6277bda79faR233). The most noticeable effect by this change is that the `line-height` style (see [_index:243](https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/scss/widgets/fluent/scheduler/_index.scss#L243)) is no longer applied causing inconsistencies in a header cell layout.

By wrapping a span element with a div element, it is easier to apply styles to the span element without affecting a header cell layout.